### PR TITLE
Create LoginPage of Luma

### PR DESCRIPTION
**Pull request by EstebanGlober**
Create The Login page of luma

- Create the attributes of page
- Create methods of page

### CODE REVIEW

File: src/main/java/pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class LoginPage {
+
+    private WebDriver driver;
+    private By emailField = By.id("email");
+    private By passwordField = By.id("pass");
+    private By buttonSingin = By.id("send2");
+
+    public LoginPage(WebDriver driver){this.driver=driver;}
+
+    public  void Writeemail (String word){
```
Comment (larry-padilla) [inline src/main/java/pages/LoginPage.java:17]: the method cannot start with a capital letter, I suggest a more telling name, perhaps, `typeEmail()`, the input value should be `userEmail`

File: src/main/java/pages/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        driver.findElement(emailField).sendKeys(word, Keys.ENTER);
+    }
+
+    public void WritePassword (String word){
```
Comment (larry-padilla) [inline src/main/java/pages/LoginPage.java:21]: the method cannot start with a capital letter, I suggest a more telling name, perhaps, `typePassword()`, the input value should be `password`

**Review by larry-padilla — CHANGES_REQUESTED** [review]

File: src/main/java/pages/LoginPage.java (hunk 1 above)
Comment (EstebanGlober) [reply src/main/java/pages/LoginPage.java:17]: DOne

File: src/main/java/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    public  void typeEmail (String userEmail){
+        driver.findElement(emailField).sendKeys(userEmail, Keys.ENTER);
+    }
+
+    public void typePassword (String password){
+        driver.findElement(emailField).sendKeys(password, Keys.ENTER);
+    }
+
+    public void ClickButtonSignIn (){
```
Comment (larry-padilla) [inline src/main/java/pages/LoginPage.java:25]: The method name cannot start with a capital letter.

**Review by larry-padilla — CHANGES_REQUESTED** [review]

File: src/main/java/pages/LoginPage.java (hunk 3 above)
Comment (EstebanGlober) [reply src/main/java/pages/LoginPage.java]: Done

**Review by larry-padilla — APPROVED** [review]

**Pull request merged**